README.md

```diff
@@ -160,6 +161,15 @@ notice($secret)
 
 This outputs `Notice: Scope(Class[main]): Sensitive [value redacted]`
 
+Note that the Hiera backend can also return a string wrapped with the Sensitive type, but you need to explicitly add a `lookup_options` configuration.
```

Comment:
This is no longer required because of https://github.com/TraGicCode/tragiccode-azure_key_vault/pull/96 as mentioned above.

Reply:
I undid this change.